packages/file_selector/file_selector_android/example/lib/get_directory_page.dart

```diff
@@ -0,0 +1,78 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
I assume the example Dart files are exact copies of the ones in the other implementation packages?

Reply:
Yes! I think they have very few changes but overall a copy. 
We didn´t include the Android version in the ´file_selector´ base package because that would need a release for it to be deployed, plus it would be modifying two packages at once.

We´ll include that in a following PR.